### ISSUE TITLE
Codelab site description

### ISSUE THREAD

**Issue by jerdog** [SolaceDev/solace-dev-codelabs-site#8]
Currently the Codelabs site has a basic "Welcome to Codelabs!" blurb which needs updated

**Comment by jerdog** [comment]
Location looks to be at `/site/app/views/default/index.html`

**Comment by jerdog** [comment]
Adding myself and @Mrc0113 to this one as we should all contribute to the blurb

**Comment by Mrc0113** [comment]
DRAFT BELOW - @jerdog  what do you think? 

## Welcome to Solace Developer Codelabs! 
This site provides tutorials that give a guided, hands-on experience with the Solace PubSub+ Platform. Each codelab will either step you through the process of building application(s) with PubSub+, learn how to use a specific PubSub+ feature, or even integrate PubSub+ with other technologies. 

Not Findings the Tutorials you want? 
[More Solace Tutorials Here! ](https://docs.solace.com/Developer-Tutorials/Developer-Tutorials.htm)
[Discuss on the Solace Community](https://solace.community)

**Comment by jerdog** [comment]
I think it's good @Mrc0113 - the only thing would be to linkify `Solace PubSub+ Platform` at the beginning. Also, the title I would instead go with "Welcome to Solace Developer Codelabs"

**Comment by Mrc0113** [comment]
I'm good with the above suggestions. I updated the draft above. 
As far as the linkify when I talked to Blaine this morning he didn't see a way to easily do that yet so we'll have to add that as a separate issue to look at later.

**Comment by jerdog** [comment]
Just add a link? You're already doing that with the "More Solace Tutorials Here!" bit?

**Comment by Mrc0113** [comment]
Agree - on the main page it looks easy. 
The main description paragraph at the top is basically just text and we didn't see a way of adding a link to it on the "Filter by type" pages. I assume the main page is the same. However, on the main page we also have that "Codelab tools on Github" link which is added below that main text and is already a link so I'm hoping it will be easy to add links at the bottom of the description (which is where I added mine).  

So basically easy to add a link at the bottom (I think unless @blainehussey tells me otherwise), but not so much in the text for now.

**Comment by jerdog** [comment]
Ok makes sense. Then let's add a PubSub+ Platform link with the others.
